scripts/scripts/fill_in_the_blanks.py

```
@@ -1,186 +1,143 @@
+    tokens = tokenize_sentence(language, text)
 
+    # Find indices and values of Word tokens
+    word_indices = [(i, t["value"]) for i, t in enumerate(tokens) if t.get("type") == "Word"]
+    questions = []
+    for idx, word in word_indices:
+        # Reconstruct the sentence with the idx-th Word replaced by blanks
+        masked_tokens = tokens.copy()
+        masked_tokens[idx] = {**masked_tokens[idx], "value": "_____"}
+        # Rebuild the sentence, preserving original spacing/punctuation
+        question_text = ""
+        prev_end = 0
```

Ruff (0.8.2)

Local variable `prev_end` is assigned to but never used

Remove assignment to unused variable `prev_end`

(F841)